### PR TITLE
adds documentation/links for exposing JMX via HTTP

### PR DESCRIPTION
**Pull request by seut**

### CODE REVIEW

File: blackbox/docs/admin/monitoring.rst [hunk 1]
```diff
@@ -149,6 +149,16 @@ Metrics can be accessed using the JMX MBean object name
 
    Defines if the node is able to process SQL statements.
 
+Exposing JMX via HTTP
+=====================
+
+The JMX metrics and also a readiness endpoint can be exposed via HTTP by using
```
Comment (andreidan) [inline blackbox/docs/admin/monitoring.rst]: I suggest you drop the "also" as it impedes the reading flow a bit

File: blackbox/docs/admin/monitoring.rst [hunk 2]
```diff
@@ -149,6 +149,16 @@ Metrics can be accessed using the JMX MBean object name
...
+the `Crate JMX HTTP Exporter`_ java agent. See the `README`_ of the `Crate JMX
+HTTP Exporter`_ repository for further information and usage details.
```
Comment (andreidan) [inline blackbox/docs/admin/monitoring.rst]: I would keep this short(er) by stopping after "information". ( "and usage details" doesn't add much value IMO)

File: blackbox/docs/admin/monitoring.rst [hunk 3]
```diff
@@ -149,6 +149,16 @@ Metrics can be accessed using the JMX MBean object name
 
    Defines if the node is able to process SQL statements.
 
+Exposing JMX via HTTP
+=====================
+
+The JMX metrics and a readiness endpoint can be exposed via HTTP by using
+the `Crate JMX HTTP Exporter`_ java agent. See the `README`_ of the `Crate JMX
```
Comment (nslater) [inline blackbox/docs/admin/monitoring.rst]: "Java"

"of the" -> "in the"

File: blackbox/docs/admin/monitoring.rst [hunk 4]
```diff
@@ -149,6 +149,16 @@ Metrics can be accessed using the JMX MBean object name
...
+HTTP Exporter`_ repository for further information.
```
Comment (nslater) [inline blackbox/docs/admin/monitoring.rst]: "for further" -> "for more"

File: blackbox/docs/admin/monitoring.rst [hunk 5]
```diff
@@ -149,6 +149,16 @@ Metrics can be accessed using the JMX MBean object name
...
+
+
```
Comment (nslater) [inline blackbox/docs/admin/monitoring.rst]: remove superfluous newline

**Pull request merged**